### ISSUE TITLE
Unable to config fallback key

### ISSUE THREAD

**Issue by Alsan** [pguedes/gesticle#2]
As in the configuration section of the readme,

```text
For example, if the active application window is gedit, and the detected gesture is a 3 finger swipe, the preferred setting will be swipe.down.3.gedit and if that is not configured will look for a swipe.down.3
```

But how can I define both key hash and table name at the same time? For example:

```toml
[swipe.left]
3 = "super+s"
  [swipe.left.3]
  codium = "ctrl+tab"
```

will cause a `Key '3' has already been defined` error.

**Comment by pguedes** [comment]
hmmm i guess that's just wrong... was pretty sure i tested this but looks like i did not... 

Quickly looked around and it seems that this is just not supported in toml (swipe.left.3 defined with value and as table with sub-values)

i guess it could be changed to support something like

```
[codium.swipe.left]
3 = "ctrl+tab"

[swipe.left]
3 = "super+s"
```

or

```
[swipe.left.3]
default = "super+s"
codium = "ctrl+tab"
```

any feelings about prefered syntax? i think i like the first (groups cfg by app)

**Comment by Alsan** [comment]
Me too, grouping by app is better.

**Comment by pguedes** [comment]
fixed in f925c7a
